### ISSUE TITLE
[Mobs resetting] Mobs not properly aggroing and they reset in middle of aggro range

### ISSUE THREAD

**Issue by Elexias** [chromiecraft/chromiecraft#7345]
### What client do you play on?

enGB

### Faction

Alliance

### Content Phase:

70

### Current Behaviour

Mobs keeps losing aggro in middle of fight. Aggro range worked properly earlier in the morning and yesterday in the noon. There was some update yesterday and they got fixed, but somehow they do not work again.

![image](https://github.com/user-attachments/assets/fa5dc7d7-a2c9-422a-baa5-e10223de52ef)
![image](https://github.com/user-attachments/assets/02184251-2bd1-404e-80a8-ca96d8a1ce63)
![image](https://github.com/user-attachments/assets/b7768a07-b48a-40c2-92a4-2fcff7970fd7)
![image](https://github.com/user-attachments/assets/24f708a6-d8ba-4136-836d-512fa10a47e9)

### Expected Blizzlike Behaviour

-

### Source

-

### Steps to reproduce the problem

Aggro lot of mobs and see

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/dccec8764408a58eb01ba85827f8686b196d370c

### Operating system

Ubuntu 22.04

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-bg-item-reward](https://github.com/azerothcore/mod-bg-item-reward)
- [mod-BreakingNewsOverride](https://github.com/AnchyDev/BreakingNewsOverride)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chat-transmitter](https://github.com/azerothcore/mod-chat-transmitter)
- [mod-cta-switch](https://github.com/azerothcore/mod-cta-switch)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna](https://github.com/azerothcore/mod-eluna)
- [mod-instanced-worldbosses](https://github.com/nyeriah/mod-instanced-worldbosses)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-low-level-rbg](https://github.com/azerothcore/mod-low-level-rbg)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-progression-system](https://github.com/azerothcore/mod-progression-system)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-rdf-expansion](https://github.com/azerothcore/mod-rdf-expansion)
- [mod-transmog](https://github.com/azerothcore/mod-transmog)
- [mod-weekend-xp](https://github.com/azerothcore/mod-weekend-xp)
- [mod-zone-difficulty](https://github.com/azerothcore/mod-zone-difficulty)
- [lua-carbon-copy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-event-scripts](https://github.com/55Honey/Acore_eventScripts)
- [lua-exchange-npc](https://github.com/55Honey/Acore_ExchangeNpc)
- [lua-level-up-reward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-recruit-a-friend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-zonecheck](https://github.com/55Honey/acore_Zonecheck)

### Customizations

None

### Server

ChromieCraft

**Comment by heyitsbench** [comment]
Keep in mind that a creature will leash (lose aggro) if they both have not been the victim of a hostile action for twelve seconds (e.g. casting a spell, melee hit, not consecrate or DoTs), are outside a small radius around their home position, and have not stood still for five seconds (i.e. to melee you).

**Comment by Elexias** [comment]
Hey, I've been farming those mobs for weeks... they are not working properly (aggro), being attacked all time nad htey keep resetting.

**Comment by Elexias** [comment]
[hromiecraft.com/en/server-status/
](https://github.com/chromiecraft/chromiecraft/issues/6689)

Dont want this to repeat again... me trying or month to convince you...

**Comment by mcblack4ever** [comment]
I’ve encountered this problem as well. While leveling an alt, I noticed that mobs don’t lose aggro. When you pull one mob from a group, the entire pack chases you, even if you run a great distance. They only lose aggro once the primary mob is defeated.

**Comment by heyitsbench** [comment]
> When you pull one mob from a group, the entire pack chases you, even if you run a great distance. They only lose aggro once the primary mob is defeated.

This is Blizzlike behavior as can be seen and described in [this video](https://www.youtube.com/watch?v=xnrUkbEbiJk&t=452s).

**Comment by Elexias** [comment]
![image](https://github.com/user-attachments/assets/010425aa-37c8-4134-9ebd-65c96b09d826)

They freeze me for 8 seconds, mana burn, and they reset :D OP Chromie

**Comment by Elexias** [comment]
> > When you pull one mob from a group, the entire pack chases you, even if you run a great distance. They only lose aggro once the primary mob is defeated.
> 
> This is Blizzlike behavior as can be seen and described in [this video](https://www.youtube.com/watch?v=xnrUkbEbiJk&t=452s).

This is vanilla wow (classic wow), not wotlk client of retail wow.

**Comment by Elexias** [comment]
You're telling me that it is normal that if you pull 10 mobs, they will reset if you dont attack them? On retail, if you pull 10 mobs, they will stab you to death even if you don't attack them for certain amount of time. Here I pull them on their OWN location, not far away, and just watch them, they will forget about me.

**Comment by Rorschach91** [comment]
@heyitsbench I think @Elexias is right. I play WoW from 2008 and this thing has never happen in wotlk or in another expansion.

**Comment by Egor-aux** [comment]
@Elexias is right. I left CC for 2 months. I used to farm the orcs outside Hellfire Citadel for gold and I know all the pulls by heart. I tried to do the same pulls I always did, and mobs reset middle fight. It's like their reset range is cut in half.

**Comment by heyitsbench** [comment]
> [...] not wotlk client of retail wow.

Then provide that as a source.

> You're telling me that it is normal that if you pull 10 mobs, they will reset if you dont attack them?

Depending on the circumstances, absolutely that is what is supposed to happen.

> I play WoW from 2008 and this thing has never happen in wotlk or in another expansion.

"this thing" is very vague and I still have yet to see something that does not line up with official servers as I myself have observed them.

> I tried to do the same pulls I always did, and mobs reset middle fight.

I hope I don't need to say this, but AC/CC have been known for extreme leash ranges and incorrect behavior on that front, so comparing current CC to old CC is obviously going to be drastically different now that anything has been improved regarding the leash system.

For the record, I want this system to be accurate. If there is a problem, I do want to fix it, but I have not observed any of this odd leash behavior anyone here has mentioned, and *I've tried to find it*.

**Comment by Monet01** [comment]
> > When you pull one mob from a group, the entire pack chases you, even if you run a great distance. They only lose aggro once the primary mob is defeated.
> 
> This is Blizzlike behavior as can be seen and described in [this video](https://www.youtube.com/watch?v=xnrUkbEbiJk&t=452s).

Thanks for sharing the video.  The issue was related to split pulling.  It was not working properly.   When I pulled a mob from a pack, side mobs didn't lose aggro when I ran far away and even when they didn't receive any damages from the player.  Side mobs would lose aggro and reset when the primary mob is defeated from the pack.

(after the latest update, it seems this issue has been fixed in ChromieCraft).

**Comment by Elexias** [comment]
> > [...] not wotlk client of retail wow.
> 
> Then provide that as a source.
> 
> > You're telling me that it is normal that if you pull 10 mobs, they will reset if you dont attack them?
> 
> Depending on the circumstances, absolutely that is what is supposed to happen.
> 
> > I play WoW from 2008 and this thing has never happen in wotlk or in another expansion.
> 
> "this thing" is very vague and I still have yet to see something that does not line up with official servers as I myself have observed them.
> 
> > I tried to do the same pulls I always did, and mobs reset middle fight.
> 
> I hope I don't need to say this, but AC/CC have been known for extreme leash ranges and incorrect behavior on that front, so comparing current CC to old CC is obviously going to be drastically different now that anything has been improved regarding the leash system.
> 
> For the record, I want this system to be accurate. If there is a problem, I do want to fix it, but I have not observed any of this odd leash behavior anyone here has mentioned, and _I've tried to find it_.

You probably fixed what you wanted (with mob pack and side mobs) but something very bad happened with pulling and resetting... And it is not 12 seconds without being damaged...(someone said that it will take 12 seconds before they get reset)...generally it is not working atm.

**Comment by Egor-aux** [comment]
https://youtu.be/kthqfk4FsSM Is this how it's suposed to work? Just to make it clear.

**Comment by Elexias** [comment]
> https://youtu.be/kthqfk4FsSM Is this how it's supposed to work? Just to make it clear.

Exactly, that is how WOW is not supposed to work and never worked. You dot him, he dies no matter if you attack him by single target spell or not. Mob should kill you even if you dont attack him. According to @heyitsbench it is normal for mob to reset in middle of attacking you for example if you afk.

**Comment by Egor-aux** [comment]
I would suggest a reverse, it's just very very annoying.
